src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-field.component.ts

```diff
@@ -81,6 +81,9 @@ import { DefaultValueOptionsComponent } from "./default-value-options/default-va
   ],
 })
 export class AdminEntityFieldComponent implements OnChanges {
+  delete() {
+    throw new Error("Method not implemented.");
+  }
```

Comment:
this is probably accidentally left over from some experiments during development?